packages/module/patternfly-docs/content/extensions/virtual-assistant/design-guidelines.md

```diff
@@ -0,0 +1,206 @@
+
+### Messages 
+
+<div class="ws-docs-content-img">
+![A basic user message, bot message, and quick reply buttons.](./img/message-elements.svg)
+</div>
+
+1. **User message:** Messages that the user has sent to the ChatBot.
+1. **Bot message:** Messages from the ChatBot, which are marked with an "AI" label to communicate the use of AI to users. You should choose a descriptive name for your ChatBot.
+1. **Avatar:** Representative image for your ChatBot and the user. ChatBot avatars should align with your product's brand and any existing brand standards.
+1. **Name:** Identifier for your ChatBot and the user. Choose a name for your ChatBot that users can easily identify as a bot. 
+1. **Label:** Labels ChatBot messages as "AI."
+1. **Timestamp:** The relative or absolute time that a message was sent.
+1. **Quick responses:** Programmable, clickable actions that allow users to quickly answer questions from the ChatBot.
+1. **Sources:** Cards that link to documentation or other external resources. When multiple sources are included, users can paginate through the different options. 
+1. **Response actions:** Actions that allow users to interact with a bot message. these typically include providing feedback, copying, sharing, or reading aloud, but [custom message actions](/patternfly-ai/chatbot/messages#custom-message-actions) are also supported.
+
+### Message bar 
+
+To message the ChatBot, users can type directly into the message bar in the footer or click any included actions. 
+
+<div class="ws-docs-content-img">
+![Elements of a message bar, including input actions.](./img/message-bar-elements.svg)
+</div>
+
+1. **Attach button:** Allows users to upload files from their computer. 
+1. **Use microphone button:** Supports speech recognition to allow users to use voice input. This feature is currently only available in Chrome and Safari.
+1. **Send button:** Allows users to send a typed message. This button should be disabled until a user has input text. 
+
+When a user chooses to use speech input via the microphone button, the button will display an animation to indicate that the ChatBot is listening to the user (as shown in [this speech recognition example](/patternfly-ai/chatbot/ui#message-bar-with-speech-recognition-and-file-attachment)). To stop voice input, users will need to select the microphone button again.
+
+<div class="ws-docs-content-img">
+![Active listening button in the message bar.](./img/listening.svg)
+</div>
+
+When a user selects the "listen" [message action](/patternfly-ai/chatbot/messages#messages-actions) on a bot message, the message contents will be read aloud and a stop button will be displayed as the only action in the message bar. Selecting this button will halt the message playback.
```

Comment:
The stop button is meant for streaming responses, like what ChatGPT has. They technically could use it for that purpose if they wanted. We don't supply any text to speech functionality.

Reply:
ohhh like it stops a message response as its happening, I see. I'm updating the image to try and convey that, but lmk if it seems wonky